libs/auth/src/selectors/register.selector.ts

```diff
@@ -0,0 +1,43 @@
+import { createSelector, MemoizedSelector } from '@ngrx/store';
+
+import { selectAuthFeatureState } from './auth-feature.selector';
+import { DaffAuthToken } from '../models/auth-token';
+import {
+  DaffAuthRegisterReducerState,
+  DaffAuthFeatureState
+} from '../reducers/public_api';
+
+export interface DaffAuthRegisterSelectors<T extends DaffAuthToken> {
+  selectAuthRegisterState: MemoizedSelector<object, DaffAuthRegisterReducerState>;
+  selectAuthRegisterLoading: MemoizedSelector<object, boolean>;
+  selectAuthRegisterErrors: MemoizedSelector<object, string[]>;
+}
+
+const createRegisterSelectors = <T extends DaffAuthToken>() => {
+  const selectAuthRegisterState = createSelector(
+    selectAuthFeatureState<T>(),
+    state => state.register
+  )
+
+  const selectAuthRegisterLoading = createSelector(
+    selectAuthRegisterState,
+    state => state.loading
+  );
+
+  const selectAuthRegisterErrors = createSelector(
+    selectAuthRegisterState,
+    state => state.errors
+  );
```

Comment:
I think that these selectors should go in a different file too. I think we should only have selectors in the main file when we don't have a choice, i.e. when they are selectors that depend on other selectors within auth.